gammapy/data/event_list.py

```diff
@@ -393,6 +393,58 @@ def peek(self):
         plt.tight_layout()
         plt.show()
 
+    def peek_time_map(self, num_sides=0):
```

Comment:
So far we've been using the convention to have all plotting functions start with `plot_`, and then to have one function `peek` that does a panel of quick-look plots. Maybe for now keep that style, and rename `peek_time_map` to `plot_time_map` here?